__mocks__/env-config.mock.ts

```diff
@@ -0,0 +1,9 @@
+import { IConfig } from "../utils/config";
```

Comment:
Alternatively to this module you can write a mock for `utils/config.ts` itself: https://jestjs.io/docs/manual-mocks#mocking-user-modules

I don't have strong opinions on this, though. Use whatever you feel better with.

Reply:
I'll leave it for refactoring if you don't mind.